docs/cli-main-commands.md

```diff
@@ -0,0 +1,117 @@
+---
+id: cli-main-commands
+title: Main commands
+sidebar_label: Main commands
+---
+
+This are general purpose commands that will help you to set up and interact with your development environment.
+
+## aragon run
+
+The `run` command takes care of completely setting up the environment needed for running your Aragon app. 
+
+```
+aragon run
+```
+
+These are all the things that running `aragon run` will do for you:
+
+1. It checks whether **IPFS** and a local **Ethereum development chain** (devchain) are running and if not it will start them. Once aragon is terminated, any IPFS or dev chain it started will also be terminated.
+2. It will **publish your app** to the local aragonPM registry running in your devchain. This step executes the `aragon apm publish` internally. You can check the options and how the command works in depth [here](cli-apm-commands.md).
+3. Once the package is published it will **create a DAO** with your app installed. If you are running your app without a Kit it will grant permissions to the first account printed in the console to perform all the actions protected by the ACL in your app.
+4. After the DAO is created it will download the [Aragon client](https://github.com/aragon/aragon), install its dependencies and start it up so you can interact with the DAO in your web browser.
+
+Available options to customize the `run` command:
+
+- `--reset`: If reset is present it will reset the devchain before running. The chain will then start from scratch and all published packages will need to be recreated.
+- `--port`: The port where the devchain will be started.
+- `--kit`: The name of the contract that will be deployed as the [DAO kit](kits-intro.md) that will be used to create your DAO. If no Kit is provided it will use a default Kit that sets up the DAO with just your app.
```

Comment:
```suggestion
- `--kit`: The name of the contract that will be deployed as the [DAO kit](kits-intro.md) that will be used to create your DAO. If no Kit is provided it will use a default Kit that sets up the DAO with just your app (`bare-kit.aragonpm.eth`).
```